src/components/NotificationCenter/index.js

```diff
@@ -58,6 +59,26 @@ const mapDispatchToProps = (dispatch, ownProps) => {
           updatedKeysInTodo: { active: false }
         }
       })
+    },
+
+    snoozeTodo: (id, todoListId, todo, numberOfDays) => {
+      dispatch({
+        type: 'DISMISS_NOTIFICATION',
+        payload: {
+          id
+        }
+      });
```

Comment:
It does not really matter here because I suppose everything re-renders at the same time, but in terms of logic I would dismiss the notification **after** the todo has been updated. Otherwise, in case of a bug, the notification would be dismissed (telling the user that it worked) while the todo would not be updated